### PR TITLE
First draft of the specification with the Truebit game instead of the snarks

### PR DESCRIPTION
**Pull request by josojo**

### CODE REVIEW

File: PlasmaResearch/plasma_batch_auction.tex [hunk 1]
```diff
@@ -304,10 +305,19 @@ \subsubsection*{Price Determination}
 \end{itemize}
 \end{lemma}
 */
-Along with this price vector, the plasma operator would also generate for a batch a list of tuples representing order-volume pairs, 
-\[[(\sigma_1, \nu_1), (\sigma_2,  \nu_2), \dots, (\sigma_N, \nu_N)]\]
-where $(\sigma_i, \nu_i)$ denotes the $i^{th}$ order along with the volume $\nu_i$ of fulfilment and $N$ is the number of orders in the batch. 
-This list will be referred to as the \emph{volume-list}\label{volumeBlock} and the lists Merkle hash will be the \emph{volume-hash}.
+Along with this price vector, the plasma operator would also share the trading volumes in a compact manner. This will be done using a mapping $m$ from the order index to the trading volume\emph{volume-list}\label{volumeBlock}. But we would only store the trading volumes for fractional fulfilled orders.
```
Comment (bh2smith) [inline PlasmaResearch/plasma_batch_auction.tex]: The \emph{volume-list} doesn't look right in the pdf here. Not exactly sure what was intended with this...

Also. This single line is actually three sentences (should be turned into three lines).

File: PlasmaResearch/plasma_batch_auction.tex [hunk 2]
```diff
@@ -304,10 +305,19 @@ \subsubsection*{Price Determination}
...
+
+\[
+    m(\sigma)= 
+\begin{cases}
+    \text{trading volume},& \text{if } \sigma \text{ is partially filled}\\
```
Comment (bh2smith) [inline PlasmaResearch/plasma_batch_auction.tex]: should \text{trading volume} be $\nu_\su$ (i.e. using the notation previously established as volume of the order)?

File: PlasmaResearch/plasma_batch_auction.tex [hunk 3]
```diff
@@ -304,10 +305,19 @@ \subsubsection*{Price Determination}
...
+    1,        & \text{if } \sigma \text{ is fully filled and the has highest limit price for one pair}\\
+    0,              & \text{is not touched} \\
+    0,              & \text{is fully filled, but not highest limit price} \\
+\end{cases}
+\]
+This mapping $m$ will be stored on the root-chain, in case of a data unavailability. In the plasma chain, we will store only the \emph{volume-hash}, that is the Merkle root of the following array:
+\[[m(\sigma_1), m(\sigma_2), \dots, m(\sigma_n)]\].
```
Comment (bh2smith) [inline PlasmaResearch/plasma_batch_auction.tex]: period needs to go on the inside of \]

File: PlasmaResearch/plasma_batch_auction.tex [hunk 4]
```diff
@@ -607,7 +609,7 @@ \section{An elaboration on the snark mechanism}
 The public key used to encrypt all orders will be denoted by $\text{key}_p$.
 In order to run the proofs the most efficient, we need to calculate a special hash:
 \begin{equation}
-h = \text{sha256}(M_R(\mathcal{B}_O),\mathcal{H}_V, \text{sha256}(\beta), M_R(\vec{p}), \text{key}_p) 
+h = \text{sha256}(sha256(\mathcal{B}_O),\mathcal{H}_V, \text{sha256}(\beta), sha256(\vec{p}), \text{key}_p)
```
Comment (bh2smith) [inline PlasmaResearch/plasma_batch_auction.tex]: Did you also want to put a sha256 onto \mathcal{H}_V? I think we should also declare a math operator for sha256. I think that I will make a bunch of these changes myself and you can look at them after...

**Comment by josojo** [comment]
This pr is on hold, as we made good progress with snarks

**Comment by josojo** [comment]
No longer needed

**Pull request closed without merging**